supertokens/utils.go

```diff
@@ -104,13 +104,13 @@ func getLargestVersionFromIntersection(v1 []string, v2 []string) *string {
 	}
 	maxVersionSoFar := intersection[0]
 	for i := 1; i < len(intersection); i++ {
-		maxVersionSoFar = maxVersion(intersection[i], maxVersionSoFar)
+		maxVersionSoFar = MaxVersion(intersection[i], maxVersionSoFar)
 	}
 	return &maxVersionSoFar
 }
 
 // MaxVersion returns max of v1 and v2
-func maxVersion(version1 string, version2 string) string {
+func MaxVersion(version1 string, version2 string) string {
```

Comment:
No need to make this capital. Instead, add a copy of this in the unittesting/testingutils.go file, and expose this via that and then use that in your tests.